proto/message_contents/ciphertext.proto

```diff
@@ -0,0 +1,20 @@
+// Ciphertext
```

Comment:
Extracted Ciphertext as well, as it is used in multiple places and it seems wrong to import xmtp_envelope for it.